FrontMen/fm-hours
# feat: page titles and favicon

# Changes

## Related issues

#45 

## Added

- Titles per pages

## Removed

N/A

## Changed

- Default title at `nuxt.config.js`
- Favicon file

## How to test

- Navigate through pages and check if titles are correct
- Check if the new favicon is set

## Screenshots

N/A
